src/main/java/be/syntra/devshop/DevshopBack/entities/Cart.java

```diff
@@ -28,13 +28,13 @@
     @JsonIgnore
     private User user;
 
-    @NotNull
+
```

Comment:
why removing the @NotNull constraint? We never want a null-value in our database. If you're referring to new users who don't have a cart yet, insert an empty list instead e.g. new ArrayList<>()

Reply:
fixed